### PR TITLE
Feat/audio

### PR DESCRIPTION
**Pull request by bartoszrud**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,4 +6,5 @@ pytest==7.4.0
 transformers==4.35.0
 diffusers==0.21.4
 torch==2.1.0
-stability-sdk==0.8.4
\ No newline at end of file
+stability-sdk==0.8.4
+pydub==0.25.1
```
Comment (BartWojtowicz) [inline requirements.txt:10]: Can we also update dependencies in `pyproject.toml`?

File: src/videopython/base/video.py [hunk 2]
```diff
@@ -20,6 +23,7 @@ class VideoMetadata:
     fps: int
     frame_count: int
     total_seconds: float
+    audio: bool = False
```
Comment (BartWojtowicz) [inline src/videopython/base/video.py:26]: It's not actually `bool`, because we load audio to this variable 🤔

File: requirements.txt (hunk 1 above)
Comment (bartoszrud) [reply requirements.txt:10]: Added

**Review by BartWojtowicz — APPROVED** [review]
Let's get this merged 🙌🏻

**Pull request merged**